gtsam/slam/dataset.h

```diff
@@ -236,13 +243,23 @@ struct SfmTrack {
   Point3 point3() const {
     return p;
   }
+  void add_measurement(pair<size_t, gtsam::Point2> m) {
+    Measurements.push_back(m);
+  }
+
+  SfmMeasurements& measurements() {
+    return Measurements;
+  }
+
 };
 
+
 /// Define the structure for the camera poses
 typedef PinholeCamera<Cal3Bundler> SfmCamera;
 
 /// Define the structure for SfM data
 struct SfmData {
+  std::vector<SfmMeasurement> Measurements;
```

Comment:
Could we use lower case `measurements` for the member name here as well?

Reply:
Could you also please document what these measurements are? I understand the one in SfmTrack, but what does the one in SfmData represent?

Reply:
it's the same: measurements within the tracks. But I'll update that in documentation